db/migrate/20170109190411_make_job_belong_to_miq_task.rb

```diff
@@ -0,0 +1,27 @@
+class MakeJobBelongToMiqTask < ActiveRecord::Migration[5.0]
+  class Job < ActiveRecord::Base;
```

Comment:
`Job` has  `type` column so we have to add
also
`self.inheritance_column = :_type_disabled `